docs(cloud-security): add Triage and Prioritize section with Runtime Prioritization Engine (#37172)
DataDog/documentation: content/en/security/cloud_security_management/triage_and_prioritize/_index.md

---
title: Triage and Prioritize
further_reading:
- link: "/security/cloud_security_management/triage_and_prioritize/runtime_prioritization_engine/"
  tag: "Documentation"
  text: "Runtime Prioritization Engine"
- link: "/security/cloud_security_management/triage_and_prioritize/severity_scoring/"
  tag: "Documentation"
  text: "Severity Scoring"
- link: "/security/security_inbox/"
  tag: "Documentation"
  text: "Review prioritized findings in the Security Inbox"
---

> Cloud Security generates findings across vulnerabilities, misconfigurations, and identity risks. Triage and Prioritize covers two related capabilities: the engine that identifies the findings truly exposing your business-critical resources, and the scoring framework that translates that judgment into a per-finding severity score you can sort, filter, and route on.

Reviewer: ```suggestion
Cloud Security generates findings across vulnerabilities, misconfigurations, and identity risks. Triage and Prioritize covers two related capabilities: the engine that identifies the findings that expose your business-critical resources, and the scoring framework that translates that judgment into a per-finding severity score you can sort, filter, and route on.
```

## Runtime Prioritization Engine

> The [Runtime Prioritization Engine][1] combines runtime observability and security data to identify the ~5% of findings truly exposing your business-critical resources. It evaluates each finding across five dimensions: reachability, exposure, exploitability, business criticality, and actionability. Available in Preview for Cloud Security Vulnerabilities; coverage for misconfigurations and identity risks is in development.

Reviewer: ```suggestion
{{< callout url=https://www.datadoghq.com/product-preview/runtime-prioritization-engine/
 btn_hidden="false" header="Join the Preview!">}}
Runtime Prioritization Engine is in Preview for Cloud Security Vulnerabilities. Use this form to request access.
{{< /callout >}}

The [Runtime Prioritization Engine][1] combines runtime observability and security data to identify the ~5% of findings that expose your business-critical resources. It evaluates each finding across five dimensions: reachability, exposure, exploitability, business criticality, and actionability.
```
Adding our standard callout here, and removing the part about future development to follow our usual guidelines.

## Severity Scoring

> [Severity Scoring][2] turns the Runtime Prioritization Engine's output into a Datadog Severity Score on each finding. For vulnerabilities, it strictly follows the [CVSS 4.0][3] algorithm, enriching the base score with temporal factors (active exploits, exploitation probability) and environmental factors (runtime context, exposure, criticality of the affected resource). For misconfigurations and identity risks, severity is computed using a likelihood × impact matrix that weighs how an adversary could abuse the finding against the damage that abuse would cause.

Reviewer: ```suggestion
[Severity Scoring][2] turns the Runtime Prioritization Engine's output into a Datadog Severity Score on each finding. For vulnerabilities, it follows the [CVSS 4.0][3] algorithm, enriching the base score with temporal factors (such as active exploits or exploitation probability) and environmental factors (such as runtime context, exposure, or criticality of the affected resource). For misconfigurations and identity risks, it computes severity using a likelihood × impact matrix that weighs how an adversary could abuse the finding against the damage that abuse would cause.
```
